packages/dmn-js-drd/test/spec/features/modeling/ElementFactorySpec.js

```diff
@@ -20,21 +18,18 @@ describe('features/modeling - create elements', function() {
   beforeEach(bootstrapModeler(emptyDefsXML, { modules: testModules }));
 
 
-  function expectElement(element, attrs) {
+  function expectShapeElement(element, attrs) {
```

Comment:
Why not just `expectShape` and `expectEdge`? 👓

Reply:
No particular reason.